DEVELOPMENT.rst

```diff
@@ -135,10 +135,13 @@ To run some server tests see `.travis.yml
 b. Running AngularJS test cases
 '''''''''''''''''''''''''''''''
 
-Run client tests by starting karma::
```

Comment:
Please reinsert the second colon, see RST defs.

Reply:
Done